docs/source/tools/trigger-service/index.rst

```diff
@@ -14,120 +14,139 @@ Trigger Service
-        //specifies the connection timeout for database connection pool.
+
+        // Timeout for database connection pool. Defaults to FIXME.
         connection-timeout = 90s
       }
 
       authorization {
-        //Sets both the internal and external auth URIs.
-        //auth-common-uri = "https://oauth2/common-uri"
 
-        // Auth Client to redirect to login , defaults to no
-        auth-redirect = "yes"
+        // Auth client to redirect to login. Defaults to `no`.
+        auth-redirect = "no"
+
+        // The following options configure the auth URIs.
+        // Either just `auth-common-uri` or both `auth-internal-uri` and `auth-external-uri` must be specified.
+        // If all are specified, `auth-internal-uri` and `auth-external-uri` take precedence.
+
+        // Sets both the internal and external auth URIs.
+        //auth-common-uri = "https://oauth2/common-uri"
 
-        //Sets the internal auth URIs (used by the trigger service to connect directly to the middleware). Overrides value set by auth-common
+        // Internal auth URI used by the trigger service to connect directly to the auth middleware.
         auth-internal-uri = "https://oauth2/internal-uri"
-        //Sets the external auth URI (the one returned to the browser). overrides value set by auth-common.
+
+        // External auth URI (the one returned to the browser).
+        // This value takes precedence over the one specified for `auth-common`.
         auth-external-uri = "https://oauth2/external-uri"
-        //URI to the auth login flow callback endpoint `/cb`. By default constructed from the incoming login request.
-        auth-callback-uri =  "https://oauth2/callback-uri"
 
-        //Optional max number of pending authorization requests. Defaults to 250.
+        // Optional. URI to the auth login flow callback endpoint `/cb`. By default it is constructed from the incoming login request.
+        // auth-callback-uri = "https://oauth2/callback-uri"
+
+        // Maximum number of pending authorization requests. Defaults to 250.
         max-pending-authorizations = 250
-        //Optional authorization timeout, defaults to 60 seconds
+
+        // Authorization timeout. Defaults to 60 seconds.
         authorization-timeout = 60s
       }
     }
 
-The above starts the Trigger Service using a number of default parameters. Most notably, the HTTP port the Trigger Service listens on which defaults to 8088.
-The above config file should list all available parameters, their defaults and descriptions.
-
-The trigger-service can also be started using cli-args as shown below, one can execute the command ``daml trigger-service --help`` to find all available parameters.
-
-.. note:: Configuration file is the recommended way to run trigger-service, running via cli-args is now deprecated
+The trigger service can also be started using command line arguments as shown below. The command ``daml trigger-service --help`` lists all available parameters.
```

Comment:
The fact that this is a list all available parameters, their defaults and descriptions has been moved above. The fact that command line parameters are now deprecated has been moved below.